getting_started/first_2d_game/01.project_setup.rst

```diff
@@ -41,7 +41,8 @@ Your project folder should look like this.
 This game is designed for portrait mode, so we need to adjust the size of the
 game window. Click on *Project -> Project Settings* to open the project settings
 window, in the left column open the *Display -> Window* tab. There, set
-"Viewport Width" to ``480`` and "Viewport Height" to ``720``.
+"Viewport Width" to ``480`` and "Viewport Height" to ``720``. You can see the 
+"Project" tab on the upper left corner.
```

Comment:
Personally I really don't think this addition is necessary; the convention of describing main menus like this is very common in all software, and since it's in the upper-left corner in many cases it will be the first place someone looks.

But if we do add this, it's a Project *menu* or *main menu*, not a *tab*:
```suggestion
"Viewport Width" to ``480`` and "Viewport Height" to ``720``. You can see the 
"Project" menu on the upper left corner.
```

Or simpler, change the whole paragraph to:
```
This game is designed for portrait mode, so we need to adjust the size of the
game window. In the upper-left corner, click on *Project -> Project Settings*
to open the project settings window, in the left column open the
*Display -> Window* tab. There, set "Viewport Width" to ``480`` and
"Viewport Height" to ``720``.
```